lib/railway_ipc.rb

```diff
@@ -14,8 +14,10 @@
 require 'railway_ipc/rabbitmq/adapter'
 require 'railway_ipc/handler'
 require 'railway_ipc/handler_store'
+require 'railway_ipc/message_encoders'
```

Comment:
Could we just use a `require_all('railway_ipc')`? 
Just a though here, but not really related to this PR's scope.

Reply:
Good question! It's a matter of personal preference, but I like being explicit about everything that being required. It acts as a bit of documentation since someone unfamiliar w/ the code would probably open this file first, they can see at a glance the different pieces.

Reply:
Cool cool. I figured that might be the case :)